src/notation/internal/notationactioncontroller.cpp

```diff
@@ -1907,6 +1920,110 @@ void NotationActionController::toggleStrike()
     currentNotationInteraction()->toggleStrike();
 }
 
+bool NotationActionController::isMasterNotation(const INotationPtr notation) const
+{
+    return globalContext()->currentMasterNotation()->notation() == notation;
+}
+
+QList<INotationPtr> NotationActionController::getOpenNotations()
```

Comment:
We give preference to `std` data types instead of `Qt` (unless it's directly related to the UI)